### DISCUSSION TITLE
Creating flow with configuration files

### DISCUSSION THREAD

**Discussion by oguzyazan-infoline** [emqx/emqx#11719] [Q&A] [answered]
I'm trying to publish some message when a client connect/disconnect. EMQX is running on the Linux server and I don't have an access to EMQX dashboard UI. Is there a way to create flow with configuration files?

**Comment by id** [comment, marked as answer]
@oguzyazan-infoline if you don't have UI but can access API on 18083 port, you could leverage REST API: https://www.emqx.io/docs/en/v5.2/admin/api-docs.html#tag/Rules/paths/~1rules/post
